### DISCUSSION TITLE
After installation

### DISCUSSION THREAD

**Discussion by cnawen** [jhassan8/crunchyroll-tizen#31] [Q&A]
Hi! I just installed the app on my Samsung TV but whenever I log in and try to watch a specific episode, the episode menu doesn't show anything. Tried on an emulator and it's the same

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/38185650/a1e60b97-4c6a-47f6-8bb9-abaf8bdd2966)

What did I do wrong? followed all the steps!

Thank you so much

**Comment by jhassan8** [comment]
Hi @cnawen! It's great to see the app being used! I would like to ask you a few questions to better understand the issue you're experiencing. Could you please let me know how you are accessing the specific item? Is it through a search, continuing from where you left off, selecting it from a list, or from the history? It would also be helpful to know which country you are in and what subtitle and audio settings you are using. 

From what I can see, it's working fine, but it might be failing from a specific location.

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/55010545/c73ed2b4-2b86-451d-a0a7-0f557b8e4720)

Also, make sure to have the application updated or use the online version so that you receive the updates automatically.

Greetings and thank you very much!

**Reply by cnawen** [reply]
Hey!! I searched for hours how to install Crunchyroll on Tizen and I was happy to find this way. I am accessing from all options. "My List" doesn't work, but the search does. It only lets me watch the first episode or pick up where I left off. It also doesn't let me advance to the next chapter since there is no "next" button. It only shows the config icon. You can find all the availables languages and subtitles, including spanish. I'm using the online version! and I'm from Chile.

![WhatsApp Image 2023-07-02 at 22 23 06](https://github.com/jhassan8/crunchyroll-tizen/assets/38185650/f429f4c2-bdc1-4a20-bbe1-d88c93781b47)

![WhatsApp Image 2023-07-02 at 22 23 06](https://github.com/jhassan8/crunchyroll-tizen/assets/38185650/2ddf229e-9d12-4262-9dc0-b0ede31b7f01)

![WhatsApp Image 2023-07-02 at 22 23 06](https://github.com/jhassan8/crunchyroll-tizen/assets/38185650/5fcc511e-82d4-4a13-9d75-0d5bef57e0bd)


Edit: I can see the episodes now! I guess I had to wait for an update
